include/seqan3/alignment/pairwise/detail/utility_traits.hpp

```diff
@@ -0,0 +1,45 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides utility traits for the configuration and execution of the alignment algorithm.
```

Comment:
The name `utility traits` does not give any insight what is in this file.
Consider renaming the file to chunk_traits.hpp or similar (even if you plan to extend this file later).

Reply:
I renamed it to type_traits. And yes it is already extended in another open PR.